source/js/kitchen.js

```diff
@@ -146,24 +162,35 @@ function loadTemplateFromURL(templateURL, callback) {
     });
 }
 
+function presenterForType(type) {
+    console.log(type);
```

Comment:
Oh, so you have this kitchen.js on your web-browser for debugging ? I was looking for a way to see `console.log` output when running from Xcode, but never found the answers.